components/Header/style.ts

```diff
@@ -32,7 +31,7 @@ export const HeaderContainer = styled.header<HeaderProps>`
   }
 `
 
-export const MenuListBox = styled.ul<HeaderProps>`
+export const MenuListBox = styled.ul<{ is_admin: boolean | undefined }>`
```

Comment:
```suggestion
export const MenuListBox = styled.ul<{ is_admin?: boolean }>`
```
해당 타입을 이처럼 물음표 키워드를 사용해서 표현하는 방식은 어떻게 생각하시나요?

Reply:
type 오류를 급하게 잡아내다보니 optional 방식을 사용 안 했네요 감사합니다!